app/scripts.babel/author.js

```diff
@@ -1,5 +1,5 @@
-init({ friends: true }, ({ friends }) => {
+init({ friends: true }, ({ observer, friends }) => {
```

Comment:
`init` is injecting data that requires asynchronous calls. No need to inject `observer`.